web/themes/custom/pineapple/templates/content/node--article--teaser.html.twig

```diff
@@ -84,10 +84,12 @@
 {{ attach_library('classy/node') }}
 <article{{ attributes.addClass(classes) }}>
     <div class="teaser-image-container">
+      {% set teaser_url = bamboo_render_image_style(content.field_a_teaser_image|field_target_entity.field_media_image.entity.uri.value, 'article_teaser_image_300_x_500') %}
+      {% set hero_url = bamboo_render_image_style(content.field_a_hero_image|field_target_entity.field_media_image.entity.uri.value, 'article_teaser_image_300_x_500') %}
```

Comment:
Move these variables into the right if and respective else so that they're only ever added if and when they're needed

Reply:
done